packages/nextjs/src/index.ts

```diff
@@ -31,24 +31,25 @@ export function createBrowserSupabaseClient<
     ? 'public'
     : string & keyof Database
 >({
+  supabaseUrl = process.env.NEXT_PUBLIC_SUPABASE_URL,
+  supabaseKey = process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY,
   options,
   cookieOptions
 }: {
+  supabaseUrl?: string;
+  supabaseKey?: string;
```

Comment:
it would have been nice to have an additional: useServiceKey?: boolean with auto-lookup like the other two env vars. This would be a super easy way to just tell the client that i want to work with the service key instead of anon key.